### DISCUSSION TITLE
华印社区 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#137] [模板讨论]
华印社区 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
2023-06-15 15:32:20
日志：Failed at 4/6 request,
Fail assert: {"re": "不允许使用|含有非法字符", "from": "content"} from failed_asserts,
Request URL: https://www.52cnp.com/k_misign-sign.html?operation=qiandao&format=global_usernav_extra&formhash={{formhash|urlencode}}&inajax=1&ajaxtarget=k_misign_topb

我密码里面没有任何符号，只有数字+小写字母，也不行

**Reply by wjf0214** [reply]
你用的不是我库里的吧？我库里是4条请求，你这是6条。看着是主库的。

**Comment by miaoxiaozhao** [comment]
多谢老大，用你的就没错了

**Comment by miaoxiaozhao** [comment]
用户：【】，用户组：【】
签到情况：【】
印币：【】，精华：【】，活跃值：【】，鲜花值：【】，技术：【】，违规：【】，积分：【】
